package-lock.json

```diff
@@ -50055,6 +50056,10 @@
 				"react-dom": "^18.0.0"
 			}
 		},
+		"packages/block-editor/node_modules/@wordpress/upload-media": {
+			"resolved": "packages/token-list",
+			"link": true
+		},
```

Comment:
This is a very strange record that shouldn't be there. Why is the `upload-media` package linked to the `packages/token-list` folder?

My PR (#68528) has this record that places the package in the top-level `node_modules` folder and links correctly:
```
"node_modules/@wordpress/upload-media": {
  "resolved": "packages/upload-media",
  "link": true
},
```